packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -319,12 +322,13 @@ export class SummarizerNode implements IRootSummarizerNode {
             // Assuming subtrees missing from snapshot are newer than the snapshot,
             // but might be nice to assert this using earliest seq for node.
             if (subtree !== undefined) {
-                child.refreshLatestSummaryFromSnapshot(
+                await child.refreshLatestSummaryFromSnapshot(
```

Comment:
**I don't think it's necessary**, but just as a note, this could be made "parallel" now that it is async. Similar to how summarize works with `await Promise.all`.

Reply:
Changed it to use Promise.all